presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -53,7 +56,7 @@ protected QueryRunner createQueryRunner()
             postgreSqlServer.close();
             postgreSqlServer = null;
         });
-        return PostgreSqlQueryRunner.createPostgreSqlQueryRunner(postgreSqlServer, CUSTOMER, NATION, ORDERS, REGION);
+        return createPostgreSqlQueryRunner(postgreSqlServer, Map.of(), Map.of(), List.of(CUSTOMER, NATION, ORDERS, REGION));
```

Comment:
- ImmutableMap and list
- a builder would be nice

Reply:
the point of this change is -- we have single method which serves all our needs (about 5 usage places) very well, no need for overload, and even less need for a builder